lessons/Early-Math/associative-property/langs/en-us/pages/assoc-prop-table.adoc

```diff
@@ -0,0 +1,52 @@
+= Associative Property Table
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+
+
+For each Circle, apply the Associative Property as many times as you can. Some Circles have several equivalent expressions, while some have none.
+
+[.FillVerticalSpace,cols=".^1a,^.^6a,^.^6a,^.^6a,^.^6a", stripes="none", options="header"]
+|===
+|	 | Analyze this Circle. | Apply the Associative Property. | Apply it again. | Apply it again?
```

Comment:
analyze

Reply:
How are you expecting us to apply the associative property? As another circle? As an expression with parentheses added?

Reply:
I honestly wonder whether this page would work better if you started with an expression and asked students to draw as many circles of evaluation as they could by applying the associative property.... and perhaps you could also ask which would make it easiest to evaluate the expression (you might want to choose your numbers differently to support that learning goal)